guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,136 @@ ifeval::["{build}" == "satellite"]
++
+. In the *MAC address* field, enter a MAC address of the host's provisioning interface. This ensures the identification of the host during the PXE boot process.
+. Ensure that {ProjectServer} automatically selects the *Managed*, *Primary*, and *Provision* options for the first interface on the host. If not, select them.
+. Click *OK* to save. To add another interface, click *Add Interface*. You can select only one interface for *Provision* and *Primary*.
+. Click the *Operating System* tab, and verify that all fields contain values. Confirm each aspect of the operating system.
+. From the *PXE Loader* list, select *Grub2 UEFI HTTP*.
+. Optional: Click *Resolve* in *Provisioning template* to check the new host can identify the right provisioning templates to use.
++
+For more information about associating provisioning templates, see xref:Configuring_Provisioning_Resources-Creating_Provisioning_Templates[].
+ifeval::["{build}" == "satellite"]
+. Click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+ifeval::["{build}" == "foreman"]
+. If you use the Katello plugin, click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+. Click *Submit* to save the host details.
++
+For more information about network interfaces, see {BaseURL}/managing_hosts/index#Adding_Network_Interfaces[Adding network interfaces].
+
+. Set the host to boot in UEFI mode from network.
+
+. Start the host.
+
+. Select *Kickstart default PXEGrub2*.
+
+This creates the host entry and the relevant provisioning settings. This also includes creating the necessary directories and files for UEFI booting the bare metal host. When you start the physical host and set its boot mode to UEFI HTTP, the host detects the defined DHCP service and starts installing the operating system from its Kickstart tree.
+
+ifeval::["{build}" == "satellite"]
+When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+ifeval::["{build}" == "foreman"]
+If you use the Katello plug-in, when the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+.For CLI Users
+
+. On {Project}, update the `grub2-efi` package to the latest version:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {packages-update} grub2-efi
+----
+
+. Enable `foreman-proxy-http`, `foreman-proxy-httpboot`, and `foreman-proxy-tftp true` features.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-proxy-httpboot true \
+--foreman-proxy-http true \
+--foreman-proxy-tftp true 
```

Comment:
Radovan, can you please confirm this installer does configure everything for UEFI HTTP booting? I had previously some check skips, I don't remember why I had these.

Reply:
Note to myself: add `--foreman-proxy-tftp-managed false` and info about this option (when to use it).

Reply:
@rplevka are you able to verify that these settings are correct?

Reply:
I just noticed that passing `--foreman-proxy-tftp-managed` doesn't install the netboot files either. This may be a problem. @lzap does HTTPBoot rely on those?

Reply:
Unfortunately yes, then disregard my remark. We are good here. @ekohl if you can articulate some Redmine issue for the future about how to improve this for the future that would be great.